Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6-clang as builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 as builder
```

Comment:
We need the `-clang` variant for cross-builds!

Reply:
the variant was temporary - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 should include clang now.